### PR TITLE
Test Certificate Details - First Name Last Name instead of other way around

### PR DESCRIPTION
**Pull request by freshking**
## Description

Reverse name order in certificate

## Link to Jira

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7823

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift [hunk 1]
```diff
@@ -8,28 +8,41 @@ import HealthCertificateToolkit
 extension Name {
 
 	var fullName: String {
-		var givenName = self.givenName
-		var familyName = self.familyName
+		return [resolvedGivenName, resolvedFamilyName].formatted()
+	}
+	
+	var reversedFullName: String {
+		return [resolvedFamilyName, resolvedGivenName].formatted()
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift:15]: I think you need an additional comma in the separator in this case

**Review by nickguendling — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift (hunk 1 above)
Comment (freshking) [reply src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift:15]: Not according to the ticket

Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift:15]: I see the comma in the ticket:

> **Expected result:**
> Last Name, First Name
> **Actual result:**
> First Name Last Name

Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift:15]: I don't see example screenshots, only the photo of the devices with the wrong formatting.

Regardless, I found that the tech spec has been updated with the new desired formatting: https://github.com/corona-warn-app/cwa-app-tech-spec/blob/7e46800f8d9b4c0f0efb09d624960a6476b2f534/docs/spec/dgc-overview-client.md#determining-the-formatted-name-of-the-person-from-a-dgc

**Review by pascal-brause — CHANGES_REQUESTED** [review]
I think Nick is right here, the comma is missing.

File: src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift (hunk 1 above)
Comment (freshking) [reply src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift:17]: Done

**Review by ArturFriesen — APPROVED** [review]

**Review by pascal-brause — APPROVED** [review]

**Review by nickguendling — APPROVED** [review]

**Pull request merged**